src/app/containers/Config/Config.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { TabBar, Tab } from 'rmwc/Tabs'
+import '@material/tabs/dist/mdc.tabs.min.css'
```

Comment:
I'm in progress on cleaning up some code, and I haven't needed to include the component CSS along with it. Have you tried without?